src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -10,7 +11,11 @@
 import net.sf.jabref.model.entry.ParsedFileField;
 
 import java.io.File;
-import java.util.*;
```

Comment:
The latest comment by @koppor in #1091 does not reflect the correct settings for Eclipse. See the comment by Simon for IntelliJ.